tests/multiphase_flow/Makefile.am

```diff
@@ -1,13 +1,24 @@
 include $(top_srcdir)/config/Make-rules
 
-EXTRA_PROGRAMS = free_falling_cyl_cib
+EXTRA_PROGRAMS = free_falling_cyl_cib rotating_barge_cib bargeGen
 
 # this test needs some extra input files, so make SOURCE_DIR available:
 free_falling_cyl_cib_CXXFLAGS = $(AM_CXXFLAGS) -DNDIM=2 -DSOURCE_DIR=\"$(abs_srcdir)\"
 free_falling_cyl_cib_LDADD = $(IBAMR_LDFLAGS) $(IBAMR2d_LIBS) $(IBAMR_LIBS)
 free_falling_cyl_cib_SOURCES = free_falling_cyl_cib.cpp
 
-tests: $(EXTRA_PROGRAMS)
+rotating_barge_cib_CXXFLAGS = $(AM_CXXFLAGS) -DNDIM=2 -DSOURCE_DIR=\"$(abs_srcdir)\"
+rotating_barge_cib_LDADD = $(IBAMR_LDFLAGS) $(IBAMR2d_LIBS) $(IBAMR_LIBS)
+rotating_barge_cib_SOURCES = rotating_barge_cib.cpp
+
+bargeGen_CXXFLAGS = $(AM_CXXFLAGS)
+bargeGen_LDADD =
+bargeGen_SOURCES = bargeGen.cpp
+
+barge2d.vertex: bargeGen
+	./bargeGen
```

Comment:
This isn't going to work for the test suite since tests are always executed in a temporary directory. We will need to generate this file for the test.